sci-calculators/qalculate-gtk/qalculate-gtk-4.6.1.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Note: bump with sci-libs/libqalculate!
+
+inherit xdg
+
+DESCRIPTION="Modern multi-purpose calculator"
+HOMEPAGE="https://qalculate.github.io/"
+SRC_URI="https://github.com/Qalculate/${PN}/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha amd64 ~arm64 ~hppa ~ppc ~sparc x86 ~amd64-linux ~x86-linux"
```

Comment:
The keywords here should be dropped to the unstable variants to fix the CI complaint.

Reply:
I am confused by the complaint. Everything is keyworded in both ebuilds and.

Reply:
The problem is that `qalculate-gtk` is keyworded `amd64` (i.e. stable) while it depends on the corresponding `libqalculate` version that is keyworded `~amd64` (i.e. unstable). The new version of `qalculate-gtk` should also have the unstable keyword since it is new, this will also resolve the CI complaint.